docs/troubleshooting/troubleshooting.md

```diff
@@ -89,6 +89,22 @@ Check connectivity between AKO Pod and Avi controller.
 
   Refer the [document](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#securitycontext-v1-core) to select the best suitable configuration.
 
+#### Static routes not getting populated in vrfcontext
+
+#### Possible Reasons/Solutions
```

Comment:
we should explain the reason also with solution . "AKO might have failed to add the static routes after leader election and as a workaround please reboot both leader and follower ako pods  by setting replica count to 0 and then reverting replica count back  in ako statefuleset."

Reply:
Have we taken care of this comment?

Reply:
Taking care of this comment is not required since, As per the discussion with Anurag in the meeting, he has asked not to mention that this should be done in this case specifically. Instead, just say something like whenever static routes are not getting populated reboot all the ako pods both in HA and non HA mode